tests/apitests/python/testutils.py

```diff
@@ -0,0 +1,18 @@
+import time
+import swagger_client
+import os
+from swagger_client.rest import ApiException
+import swagger_client.models
+from pprint import pprint
+
+def GetProductApi(username, password, harbor_server= os.environ["HARBOR_HOST"]):
```

Comment:
We'll need flexibility for authenticating API via approaches other than basic auth.